admin/threads-seq

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+# admin/thread-seq N
+#
+# This program prints a sequence of N integers for multithreaded
+# performance testing.  The integers are numbers of threads to
+# be sampled in a test.  The goal is to assist in graphing
+# how per-thread throughput relates to concurrency.
+#
+# The sequence is (at most) length N, starts at 2, includes the
+# number of CPU cores, and ends at 1.5 the number of CPU cores.
+# It does not have repeated items.
+#
+# We exceed the number of cores specifically to see the
+# "elbow" in the graph, when the number of threads tested
+# exceeds the number of cores.  (This is good because otherwise
+# -- assuming the software under test is perfectly scalable --
+# the graph would be a straight line parallel with the x axis.)
+
+import os, sys, math, itertools
```

Comment:
I'd suggest that this could just as well be a little Rust script, which would be a little nicer IMO. Not a blocker by any means.

Reply:
agreed, the more Python there is the more it's tempting to think about formatting, linting, Python versions, etc.

Reply:
RiiR'd 🦀